lib/etl/data/bad_group_array.json

```diff
@@ -0,0 +1,17 @@
+[
+	{
```

Comment:
It would be good to have the same indentation as in the other files here (and the files below).